src/components/AutoCompleteInput/AutoCompleteInput.js

```diff
@@ -22,6 +22,7 @@ const ItemsWrapper = styled('div')`
   height: 0px;
   overflow: visible;
   margin-top: ${props => props.theme.spacings.bit};
+  z-index: 20;
```

Comment:
Can you add this value to the theme, please? https://github.com/sumup/circuit-ui/blob/master/src/themes/default.js#L228

Reply:
Oh, I wasn't aware we already added this.  Thanks.

Reply:
I'm always one step ahead